go.mod

```diff
@@ -0,0 +1,49 @@
+module github.com/che-incubator/che-workspace-crd-operator
+
+go 1.12
```

Comment:
Note: the current PR is incompatible with Go 1.13, and apparently setting the go version to 1.12 does nothing to ensure compatibility. To build the project, I had to add 
```
replace github.com/openshift/api v3.9.1-0.20190424152011-77b8897ec79a+incompatible => github.com/openshift/api v0.0.0-20190424152011-77b8897ec79a
```

(Note this is a no-op change in Go 1.12, as both defined versions resolve to an identical snapshot of the package).

### Background
Go 1.13 changes the requirements for semantic import versioning; See section "Version validation" in the [release doc](https://golang.org/doc/go1.13#modules). Attempting to build this PR with Go 1.13.5 gives the error:
```
go: github.com/openshift/api@v3.9.1-0.20190424152011-77b8897ec79a+incompatible: invalid pseudo-version: preceding tag (v3.9.0) not found
```
This dependency comes in from the tree
```
github.com/operator-framework/operator-sdk v0.12.1-0.20191112211508-82fc57de5e5b
  github.com/operator-framework/operator-lifecycle-manager v0.0.0-20190605231540-b8a4faf68e36
    github.com/openshift/api v3.9.1-0.20190424152011-77b8897ec79a+incompatible
```
which is also built off Go 1.12, and so does not encounter the issue (building any of those snapshots with Go 1.13 fails for the same reason). 

To resolve the issue, I added the line
```
replace github.com/openshift/api v3.9.1-0.20190424152011-77b8897ec79a+incompatible => github.com/openshift/api 77b8897ec79a
```
(as suggested in the doc) and ran `go mod tidy`.

Reply:
> Note: the current PR is incompatible with Go 1.13,

It's kind of strange =) Since I built it on `go1.13.6`
Thanks for the good analyze. Will take a look closer

Reply:
Thanks for figuring it out. I wonder why it works for me on go1.13.6 without proposed changes, but it totally makes sense to apply proposed changes and be buildable on more go versions.